numpy/core/_add_newdocs.py

```diff
@@ -3938,18 +3938,19 @@
     Construct Python bytes containing the raw data bytes in the array.
 
     Constructs Python bytes showing a copy of the raw contents of
-    data memory. The bytes object can be produced in either 'C' or 'Fortran',
-    or 'Any' order (the default is 'C'-order). 'Any' order means C-order
-    unless the F_CONTIGUOUS flag in the array is set, in which case it
-    means 'Fortran' order.
+    data memory. The bytes object can be produced in C-order by default.
+    This behavior is controlled by the order parameter.
 
     .. versionadded:: 1.9.0
 
     Parameters
     ----------
-    order : {'C', 'F', None}, optional
-        Order of the data for multidimensional arrays:
-        C, Fortran, or the same as for the original array.
+    order : {'C', 'F', 'A', 'K'}, optional
+        Controls the memory layout of the copy. 'C' means C-order,
```

Comment:
```suggestion
        Controls the memory layout of the bytes object. 'C' means C-order,
```
A slight modification to the original since this is for `tobytes` and not `copy`.

Also note that this suggestion might push the line length over 80-char (it's difficult to tell in the GH interface). If so, please make sure to re-do the line breaks so that line length remains <80 chararacters.

Reply:
Actually, it is 74 chars with your suggestion. So, we would be good here.